infra/ansible/roles/django/tasks/main.yml

```diff
@@ -272,6 +272,44 @@
     enabled: yes
     daemon_reload: yes
 
+# scheduled task: watch account creation hourly
+
+- name: Copy Tournesol API watch-account-creation-1h service
+  template:
+    dest: /etc/systemd/system/tournesol-api-watch-account-creation-1h.service
+    src: tournesol-api-watch-account-creation-1h.service.j2
```

Comment:
I need to check, but I think we can use the same template for the 1h and 24h service.

Instead of having two .service.j2 file in Ansible, we could have only one using a template variable for `{{ since_n_hours }}`.